drive/drive-space.go

```diff
@@ -46,6 +46,7 @@ type DriveSpace struct {
 	SpaceType DriveSpaceType
 	RootCid   string
```

Comment:
RootCid is the root folder? For example "Ehsan" in a folder structure of: Ehsan->Photos->Summer2020

Reply:
@ehsan6sha  Each `UserDrive` has two `DriveSpace`s (PublicSpace and PrivateSpace). Each space keeps it's own rootDir CID and a pointer to it's parent drive. All of changes happen on the spaces. After you are done changing a driveSpace (with methods like write, mkdir, ...) you should call DriveSpace.Save() which will update the parent drive (according the spaceType).
After all of it, when you are done with the drive, you call UserDrive.Publish() which publishes the drive on the DriveStore